app/master/build.py

```diff
@@ -452,8 +464,13 @@ def _create_build_artifact(self):
         self._build_artifact = BuildArtifact(self._build_results_dir())
         self._build_artifact.generate_failures_file()
         self._build_artifact.write_timing_data(self._timing_file_path, self._read_subjob_timings_from_results())
-        self._artifacts_archive_file = app.util.fs.tar_directory(self._build_results_dir(),
-                                                                 BuildArtifact.ARTIFACT_TARFILE_NAME)
+        self._artifacts_zip_file = app.util.fs.zip_directory(self._build_results_dir(),
+                                                             BuildArtifact.ARTIFACT_ZIPFILE_NAME)
+        # Temporarily move aside zip file so we can create a tar file, then move it back.
+        temp_zip_path = shutil.move(self._artifacts_zip_file, tempfile.mktemp())
```

Comment:
This is a bit gross but I'm okay with it since we'll ideally remove the tar file at some point in the future.